### DISCUSSION TITLE
i have an issue that only 1 user is able to access my site at a given time!!!

### DISCUSSION THREAD

**Discussion by joe-bro-1** [vercel/community#1346] [Help]
## i have an issue that only 1 user is able to access my site at a given time!!!
i have 2 vercel.com subdomains & cannot have more than 1 person on the site even though they’re 2 different subdomains

https://user-images.githubusercontent.com/117040706/213613067-1489334e-f32b-4c3e-9aa8-a11c234b6c3b.mov

 if anyone can help it would be helpful.

**Comment by amyegan** [comment]
I have not seen this issue before, but I'm going to move your discussion over to the general Vercel Community. Maybe someone else else has seen it and can offer their solution. 🤔 

This looks like it could be an implementation issue. You should also reach out to the community of whatever dictation tool you are using. People more familiar with that tool may have some advice to share.

**Comment by joe-bro-1** [comment]
im using the web speech api
